idl/response/foxsi_get_output_image_cube.pro

```diff
@@ -302,6 +310,37 @@ for layer = 0, n_elements(source_map_spectrum) - 1 do begin
   output_map_cube = append_arr(output_map_cube, this_map, /no_copy)
 endfor
 
+;
+; Check to see if we've exceeded any of the thresholds (per detector or per pixel).
+; If so, print a warning and encourage attenuator use!
+;
+
+total_data = total(output_map_cube.data, 3)
```

Comment:
It's a bit of an edge case, but this code doesn't take into account someone supplying an image cube with just one frame.  Because `foxsi_get_output_2d_image` (currently) has no effective-area awareness, using `foxsi_get_output_image_cube` for a single frame is not crazy.